src/main/java/space/obminyashka/items_exchange/dao/UserRepository.java

```diff
@@ -52,4 +52,9 @@ public interface UserRepository extends JpaRepository<User, UUID> {
     @Modifying
     @Query("update User u set u.oauth2Login = true where u.email = :email and u.oauth2Login is null")
     void setOAuth2LoginToUserByEmail(String email);
+
+    @Transactional
+    @Modifying
+    @Query("update User u set u.validatedEmail = true where u.id = :id")
+    void setValidatedEmailToUserById(UUID id);
```

Comment:
We shouldn't expose users' ID outside. That method have to work with EmailConfirmationCode ID instead